doc/classes/GIProbe.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GIProbe" inherits="VisualInstance" category="Core" version="3.2">
 	<brief_description>
+		Real-time global illumination probe.
```

Comment:
maybe add "(GI)" after global illumination, so that the acronym found in the node name is made clear once.